data/maps/MauvilleCity_BikeShop/scripts.inc

```diff
@@ -1,203 +1,204 @@
 MauvilleCity_BikeShop_MapScripts:: @ 820EBBB
 	.byte 0
 
-MauvilleCity_BikeShop_EventScript_20EBBC:: @ 820EBBC
+MauvilleCity_BikeShop_EventScript_Rydel:: @ 820EBBC
 	lock
 	faceplayer
-	goto_if_set FLAG_RECEIVED_BIKE, MauvilleCity_BikeShop_EventScript_20EC94
-	goto_if_set FLAG_DECLINED_BIKE, MauvilleCity_BikeShop_EventScript_20EBF7
-	msgbox MauvilleCity_BikeShop_Text_20EE22, MSGBOX_DEFAULT
-	msgbox MauvilleCity_BikeShop_Text_20EE99, MSGBOX_YESNO
+	goto_if_set FLAG_RECEIVED_BIKE, MauvilleCity_BikeShop_EventScript_AskSwitchBikes
+	goto_if_set FLAG_DECLINED_BIKE, MauvilleCity_BikeShop_EventScript_SkipGreeting
+	msgbox MauvilleCity_BikeShop_Text_RydelGreeting, MSGBOX_DEFAULT
+	msgbox MauvilleCity_BikeShop_Text_DidYouComeFromFarAway, MSGBOX_YESNO
 	compare VAR_RESULT, 1
```

Comment:
Will do. Didn't even know we had them, and I had avoided defining ones because it's not the same value as other Yes/No menus. I'll fix the ones in this PR and then just fix all of the compares for `MSGBOX_YESNO` in another PR.